### PR TITLE
Improving the details about testing an org level endpoint

### PR DESCRIPTION
**Pull request by amilam**
I was going through the doc related to testing APIs and noticed these minor improvements.

### CODE REVIEW

File: en/docs/testing/test-rest-endpoints-via-the-openapi-console.md [hunk 1]
```diff
@@ -15,18 +15,14 @@ Follow the steps below to test a REST endpoint via the integrated OpenAPI Consol
...
-        - If you have set the **Network Visibility** of the endpoint in the **endpoints.yaml** to **Public**, go to step 7. 
-        - If you have set the **Network Visibility** of the endpoint in the **endpoints.yaml** to **Organization**, the GraphQL endpoint will not be invokable from outside the organization. In this instance, for testing purposes, Choreo allows you to generate a temporay test URL that will be active for 15 minutes. Execute step 6 to generate the temporary URL. 
+   !!! note - The **Network Visibility** of an endpoint is set in the **endpoints.yaml**. You can set it to **Public**, **Organization**, or **Project**. - If you have set the **Network Visibility** of the endpoint in the **endpoints.yaml** to **Public**, go to step 7. - If you have set the **Network Visibility** of the endpoint in the **endpoints.yaml** to **Organization**, the endpoint will not be invokable from outside, i.e., not publicly accessible. In this instance, for testing purposes, Choreo allows you to generate a temporay test URL that will be publicly accessible (secured with Oauth2.0) and active for 15 minutes. Execute step 6 to generate the temporary URL.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/testing/test-rest-endpoints-via-the-openapi-console.md:18]: Inline with the [Google style guide](https://developers.google.com/style/abbreviations#dont-use), We don't use the abbrevation i.e. Instead we use "that is" or "for example"

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/testing/test-rest-endpoints-via-the-openapi-console.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amilam) [reply en/docs/testing/test-rest-endpoints-via-the-openapi-console.md]: Ack. I couldn't think of a better way to replace "i.e.". Hence slightly modified that part. Fixed the indentation which I wasn't aware to break the rendering. I have now locally built and verified rendering.

File: en/docs/testing/test-rest-endpoints-via-the-openapi-console.md [hunk 2]
```diff
@@ -15,18 +15,14 @@ Follow the steps below to test a REST endpoint via the integrated OpenAPI Consol
...
 
-6. If the **Network Visibility** of the endpoint is set to **Organization**, click on **Generate URL** to generate a temporary test URL that will be active for 15 minutes. Else, skip this step. 
+6. If the **Network Visibility** of the endpoint is set to **Organization**, click on **Generate URL** to generate a temporary test URL that will be active for 15 minutes. Else, skip this step.
 
-    !!! note
-        The temporary test URL will be available **only for 15 minutes** and it will get expired after. You can deactivate the test URL by clicking on the **Deactivate URL** button.
+   !!! note
+   The temporary test URL will be available **only for 15 minutes** and it will get expired after. You can deactivate the test URL by clicking on the **Deactivate URL** button.
```
[the remark this answers is not in the document]
Comment (amilam) [reply en/docs/testing/test-rest-endpoints-via-the-openapi-console.md]: Fixed it.

File: en/docs/testing/test-rest-endpoints-via-the-openapi-console.md [hunk 3]
```diff
@@ -15,18 +15,14 @@ Follow the steps below to test a REST endpoint via the integrated OpenAPI Consol
...
 
-
-    ![OpenAPI Console](../assets/img/testing/openapi-console-org.png){.cInlineImage-full}
+   ![OpenAPI Console](../assets/img/testing/openapi-console-org.png){.cInlineImage-full}
```
[the remark this answers is not in the document]
Comment (amilam) [reply en/docs/testing/test-rest-endpoints-via-the-openapi-console.md]: Fixed it.

**Review by shaniR — APPROVED** [review]

**Pull request merged**